dashboard/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
@mDibyo yes, ECL-2.0 since we're doing this work for UC Berkeley. Otherwise we would have to go through a legal process to get the license changed.